ChatSecure/Classes/View Controllers/OTRRoomOccupantsViewController.swift

```diff
@@ -417,7 +417,7 @@ open class OTRRoomOccupantsViewController: UIViewController {
         connection?.readWrite({ (transaction) in
             room.save(with: transaction)
         })
-        refreshNotificationSwitch(notificationSwitch, room: room, animated:true)
+        //refreshNotificationSwitch(notificationSwitch, room: room, animated:true)
```

Comment:
I prefer not to leave stray commented/dead code hanging around. It looks like this can safely be removed though 👍

Reply:
Yep, the only reason I left it was just in case it made it easier to verify. I'll remove it.